src/dsp/pdr.c

```diff
@@ -2356,19 +2356,18 @@ int pldm_entity_association_pdr_remove_contained_entity(
 	num_children -= 1;
 	pldm_msgbuf_insert(dst, num_children);
 	//Add all children of original PDR to new PDR
```

Comment:
Update the comment to "Add all children of original PDR to new PDR except the removed entity one"